src/client.h

```diff
@@ -392,6 +397,7 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	void handleCommand_HudAdd(NetworkPacket* pkt);
 	void handleCommand_HudRemove(NetworkPacket* pkt);
 	void handleCommand_HudChange(NetworkPacket* pkt);
+	void handleCommand_HudSetAbove(NetworkPacket* pkt);
```

Comment:
"set above" is

* incorrect english
* too vague

maybe something like `HudReorder` or `HudSetDrawOrder`